spec/jobs/job_helpers/s3_helper_spec.rb

```diff
@@ -83,4 +83,63 @@
       expect(s3_helper.download(url).encoding.name).to eq('ASCII-8BIT')
     end
   end
+
+  let(:vaild_bucket_name) { 'valid-attempts-api-s3-bucket' }
```

Comment:
```suggestion
  let(:valid_bucket_name) { 'valid-attempts-api-s3-bucket' }
```

(plus rename the usages below)

![typo](https://media.giphy.com/media/lALXmzceZjqI1ogAYQ/giphy.gif)

Reply:
But - it's the bucket that's been sent to Vail, CO - i.e. "vail'd" 😉 😉 😉 

Leave it to me to fix someone else's typo and then pay it forward with another typo! 😅